Platform/Xiaomi/sm8250/pipa.dsc

```diff
@@ -0,0 +1,28 @@
+[Defines]
+  VENDOR_NAME                    = Xiaomi
+  PLATFORM_NAME                  = pipa
+  PLATFORM_GUID                  = 28f1a3bf-193a-47e3-a7b9-5a435eaab2ee
+  PLATFORM_VERSION               = 0.1
+  DSC_SPECIFICATION              = 0x00010019
+  OUTPUT_DIRECTORY               = Build/$(PLATFORM_NAME)
+  SUPPORTED_ARCHITECTURES        = AARCH64
+  BUILD_TARGETS                  = DEBUG|RELEASE
+  SKUID_IDENTIFIER               = DEFAULT
+  FLASH_DEFINITION               = Platform/Qualcomm/sm8250/sm8250.fdf
+  DEVICE_DXE_FV_COMPONENTS       = Platform/Xiaomi/sm8250/pipa.fdf.inc
+
+!include Platform/Qualcomm/sm8250/sm8250.dsc
+
+[BuildOptions.common]
+  GCC:*_*_AARCH64_CC_FLAGS = -DENABLE_SIMPLE_INIT
+
+[PcdsFixedAtBuild.common]
+  gQcomTokenSpaceGuid.PcdMipiFrameBufferWidth|1800
+  gQcomTokenSpaceGuid.PcdMipiFrameBufferHeight|2880
+
+  # Simple Init
+  gSimpleInitTokenSpaceGuid.PcdGuiDefaultDPI|309
+
+  gRenegadePkgTokenSpaceGuid.PcdDeviceVendor|"Xiaomi"
+  gRenegadePkgTokenSpaceGuid.PcdDeviceProduct|"Mi Pad 6"
```

Comment:
I think here the product name should be matching with mi pad5? 
[https://github.com/edk2-porting/edk2-msm/blob/8d7714e4cf236102536ffce7b36c0058cb8676a8/Platform/Xiaomi/sm8150/nabu.dsc#L30](https://github.com/edk2-porting/edk2-msm/blob/8d7714e4cf236102536ffce7b36c0058cb8676a8/Platform/Xiaomi/sm8150/nabu.dsc#L30)

Reply:
Well I guess "Mi Pad 6" looks better